### ISSUE TITLE
Trx File Fails to Publish (TFS 2015 vNext build) if there was a failed unit test

### ISSUE THREAD

**Issue by bdbvb** [mmanela/chutzpah#587]
The Chutzpah-generated trx file can be published OK in a TFS 2015 build as long as there are no failing unit tests. A failing unit test causes a trx file to be generated that fails to get published. I'm guessing this is because of some namespaces in the error message xml that the TFS build step does not like.

The error is:

Failed to publish test results: 'ns' is an undeclared prefix.

**Comment by mmanela** [comment]
Thanks, have you tried to edit the file to make it publish? What is the minimal change?

**Comment by mcnallys** [comment]
Ran into this and discovered the fix.

in vstst.cs  on the object "OutputTypeErrorInfo"  change Message from an object to a string. Leaving it as an object makes the xml serializer add the extra type information and namespaces.

I'd submit a pull but I had to make some other minor changes related to the servicestack.net package to get it to build.

https://github.com/PatriotExcalibur/chutzpah/commit/02a6f23e240b6bf50dcbd6acfb7385250fdee0eb
